hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestMetaCache.java

```diff
@@ -459,6 +460,28 @@ public void testUserRegionLockThrowsException() throws IOException, InterruptedE
 
       assertTrue(client1.getException() instanceof LockTimeoutException
         ^ client2.getException() instanceof LockTimeoutException);
+
+      // obtain the client metrics
+      MetricsConnection metrics = conn.getConnectionMetrics();
+      long queueCount = metrics.getUserRegionLockQueue().getCount();
+      assertEquals("Queue of userRegionLock should be updated twice. queueCount: " + queueCount,
```

Comment:
You don't have to add the variables to the strings in all the `assertEquals` calls because the assert message will gives the expected and actual values for `assertEquals`. It was only in `assertTrue` that I thought it would be useful to have the actual value included there, since the assert message will only say "expected true, actual false". Of course it probably doesn't hurt anything to have these here, it's only extra code.